src/meshapi/types/uisp_api/data_links.py

```diff
@@ -0,0 +1,144 @@
+# Auto-generated by ChatGPT from the UISP API Swagger file available here
+# https://uisp.mesh.nycmesh.net/nms/api-docs/swagger.json
+# with small manual tweaks to correct its mistakes. If these are wrong don't @ me
```

Comment:
The correctness of these types is impossible to automatically test without _extensive_ work. The UISP devs themselves may not even be able to do it easily

That being said, the consequence for having wrong types here is extremely minor. You'll get an annoying warning from mypy about a property not existing (which will be contradicted by the UISP docs). But this issue is actually guaranteed to be the case at some point for at least some properties (since UISP is likely to add new fields to their API in the future). The way to get unblocked would just be to add the missing type (as documented by Ubiquiti) and the mypy error will go away

These types aren't even used for runtime validation, they're really just a connivence feature for devs (and a way to make mypy complain less). If you really don't want them in here, my fallback plan would be to just mark everything as `dict` and give up on type definitions for UISP responses. On the whole I think keeping the ChatGPT types leans into more benefit than harm, but I do see where you are coming from in not trusting the AI farther than you can unit test it.

I marked this file as excluded from the codecov reports because it was juicing our stats, but FWIW all the lines were green (though this just indicates they are imported _somewhere_, not that useful)